govcd/api_vcd_test.go

```diff
@@ -128,8 +130,9 @@ type TestConfig struct {
 		OVAChunkedPath string `yaml:"ovaChunkedPath,omitempty"`
 	} `yaml:"ova"`
 	Media struct {
-		MediaPath string `yaml:"mediaPath,omitempty"`
-		Media     string `yaml:"mediaName,omitempty"`
+		MediaPath       string `yaml:"mediaPath,omitempty"`
+		Media           string `yaml:"mediaName,omitempty"`
+		PhotonOsOvaPath string `yaml:"photonOsOvaPath,omitempty"`
```

Comment:
Fixed

Reply:
Is there any way we could avoid adding one more parameter to the test config? Why does LB test explicitly need Photon? Wouldn't other OVAs with VMware Tools (setup correctly) work?

I'm thinking that either way most of us should be using a capable OVA for the tests, so I suggest we skip or fail LB test if the OVA is not capable enough as opposed to adding one more parameter.

@dataclouder @vbauzysvmware ?

Reply:
As far as the OVA capability - inside the test there is a script which relies on guest customization, python3, specific firewall disable script and being able to set IPs properly. Not all OSes do it in exactly the same way. It would be overkill to identify OSes if an OS supports all that.

Reply:
Changed to catalogItem as agreed. Now it checks if catalogItem is Photon OS. If it's not - just skips the test.

Reply:
Neat!